conda_requirements.txt

```diff
@@ -7,3 +7,5 @@ pandas>=0.18.1
 scikit-learn==0.18.1
 seaborn==0.7.1
 ipython==5.1.0
+python-wget=3.2 
```

Comment:
This lib is a wget-to-python catch that can be found on this site https://anaconda.org/bioconda/python-wget. Is used to do file downloads.